packages/airnode-validator/src/config/config.test.ts

```diff
@@ -1,8 +1,21 @@
 import { readFileSync } from 'fs';
 import { join } from 'path';
 import { ZodError } from 'zod';
-import { Config, chainOptionsSchema, configSchema, nodeSettingsSchema, ChainOptions, NodeSettings } from './config';
+import { zip } from 'lodash';
```

Comment:
```suggestion
import zip from 'lodash/zip';
```
although I guess this doesn't matter as it's a test file. I still haven't been able to find anything like an ESLint rule that could ban certain phrases like `from 'lodash';` 😞